### DISCUSSION TITLE
타입스크립트에서 자바스크립로 컨버팅하는 과정이 오래걸리는데 해결하는 법 없나요?

### DISCUSSION THREAD

**Discussion by zeze103** [naverz/zepeto-studio-kor#1168] [5. Q&A]
처음 개발할때는 문제 없었는데 점점 시간이 지날수록 컨버팅 시간이 길어지다보니
지금은 5분에서 10분 사이 걸리고 어쩔때는 15분 이상 걸리면서 에러가 발생됩니다.

Curl error 28: Operation timed out after 600008 milliseconds with 0 bytes received
에러 발생시 코드가 프로젝트에 정상 적용이 안되서, 따로 Force Rebuild 하여 적용하였습니다.

코드 작성은 빈공간을 추가해서 똑같았습니다.
이런 현상이 나타난것 한달쯤 된것 같습니다. (에러 발생하는건 최근 패키지 업데이트 이후)

![캡처](https://user-images.githubusercontent.com/31619436/184314566-6eb80d89-c76f-4aa7-8aeb-c4033363ccc2.PNG)

**Comment by zlmrzlo** [comment]
컨버팅 빠른 버전으로 새로운 프로젝트를 만들고 거기에 옮겨놓고 작업을 하는데도 계속 이런 문제가 발생한다는 이야기인가요?
용량이 많아서 그런 것은 아닐까요?

**Reply by zeze103** [reply]
가장 큰 용량의 에셋 중 안쓰는 에셋을 지우니 해결되었습니다;;
그것도 파일 하나마다 용량이 클 수록 컨버팅에 연관이 있는것 같습니다 ㅎ;;

그런줄 알았는데 몇번 잘되다가 다시 원래대로 컨버팅 시간이 길어지네요;;

**Comment by Cha-SeongHun** [comment]
동의합니다. 간단한 코드 수정에도 10분 이상 컴파일하는 경우가 잦아 모듈지원 등 빌드타임을 감소할 수 있는 방법으로 개선되었으면 좋겠습니다.
